.vscode/settings.json

```diff
@@ -1,7 +1,22 @@
 {
-    "python.testing.pytestArgs": [
-        "."
+    "python.linting.pylintEnabled": true,
```

Comment:
It looks like ``pylint`` isn't in the ``poetry`` dev dependencies btw. Do we want to enable it or not?

I suggest we either don't use this linter or add it to the dependencies.

@dalonsoa @davidorme

Reply:
If we want to use it - and it seems we want, right @davidorme - it should be in the dependency list, yes.